### PR TITLE
Add upstream builds to website (v2)

### PR DESCRIPTION
**Pull request by jerboaa**
Follow up for:
https://github.com/AdoptOpenJDK/openjdk-website/pull/446/

Specifically commit: 56c393c

### CODE REVIEW

File: src/scss/styles-12-upstream.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+table#release, table#ea {
+  margin-left: 0;
+  margin-right: 0;
+  font-size: small;
+  border-collapse: collapse;
+  text-align: center;
+}
+
+table#release td, table#ea td {
+  padding: 5px 0px 10px 2px;
```
Comment (gdams) [inline src/scss/styles-12-upstream.scss]: please use em rather than px.

**Comment by gdams** [comment]
staging link: https://staging.adoptopenjdk.net/447/upstream.html

**Comment by karianna** [comment]
I'd almost change ready for use to "Official Release"

**Comment by jerboaa** [comment]
> I'd almost change ready for use to "Official Release"

Sure, either phrasing works for me. Thanks!

**Comment by jerboaa** [comment]
Closes #441

**Pull request merged**